### PR TITLE
Feature/tm-10, tm-13

### PR DESCRIPTION
**Pull request by Gallenmar**

### CODE REVIEW

File: app/edit/[id]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { Card, Container, Stack, Typography } from '@mui/material';
+import { Task, useTaskContext } from '../../context/TaskContext';
+import { useRouter } from 'next/navigation';
+import TaskForm from '../../components/TaskForm';
+import { useEffect, useState } from 'react';
+
+export default function Edt({ params }: { params: { id: string } }) {
+  const { id } = params;
+  const { tasks, addOrEditTask } = useTaskContext();
+  const router = useRouter();
```
Comment (wanpii) [inline app/edit/[id]/page.tsx]: doens't seem like you're using router anywhere in this component?

File: app/tasks/[id]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	useEffect(() => {
+		const foundTask = tasks.find((t) => t.id === id);
+		if (foundTask) {
+			setTask(foundTask);
+		}
+	}, [id, tasks]);
+
+	if (!task) {
+		return (
+			<Container maxWidth="sm">
```
Comment (wanpii) [inline app/tasks/[id]/page.tsx]: indentation is crazy again :D

File: app/tasks/[id]/page.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+						{task.title}
+					</Typography>
+
+					<Typography variant="body1">
+						<strong>Description:</strong>{" "}
+						{task.description || "No description provided"}
+					</Typography>
+
+					<Typography variant="body1">
+						<strong>Type:</strong> {task.type || "No type assigned"}
```
Comment (wanpii) [inline app/tasks/[id]/page.tsx]: use ?? instead of ||

Comment (Gallenmar) [reply app/tasks/[id]/page.tsx]: fixed

**Review by wanpii — APPROVED** [review]

**Pull request merged**